### DISCUSSION TITLE
Problem with CSS, favicon, files and script in the site creations

### DISCUSSION THREAD

**Discussion by Jorge-CB** [globaleaks/globaleaks-whistleblowing-software#3455] [Q&A]
Hello,

I have installed Version 4.11.5.

I have created a second site.

The new channels, questionnaires, users and the logo change, from the second site, work.

But the new CSS, favicon that I have uploaded through the web interface, do not work, neither does the script that I have copied in the folder /var/globaleaks/scripts/ and whose name is '2'.

What I can be doing wrong?

Thanks in advance.

**Comment by evilaliv3** [comment]
Hello @Jorge-CB 

I suspect that in your tests you could have been accessing the secondary tenants via the root tenant domain. Is this the case?

For security reasons CSS and javascripts are loaded on secondary tenants only when they are accessed their own dedicated domain name only.

**Reply by Jorge-CB** [reply]
Thank you very much Giovanni.

I have Globaleaks in a virtual machine with which I am doing tests.

I am accessing my site 2 with the URL: http://XXX.XX.X.XX/t/s2/#/
![image](https://github.com/globaleaks/GlobaLeaks/assets/132654228/250b5751-d87f-4fff-9e41-029683a0bec4)


My site 1 URL is: http://XXX.XX.X.XX/#/

How can I access site 2 with its own dedicated domain name? Where is it configured?

Thanks in advance.

**Comment by evilaliv3** [comment]
You need to configure an hostname in the network settings of the site and for security reasons als enable a dedicated https certificate for the tenant.
